Remove unused header that doesn't exist on older OS Xes.

--- ld64-127.2/src/ld/parsers/libunwind/AddressSpace.hpp.orig	2015-01-27 23:24:49.000000000 +0100
+++ ld64-127.2/src/ld/parsers/libunwind/AddressSpace.hpp	2015-01-27 23:26:21.000000000 +0100
@@ -37,7 +37,6 @@
 #include <mach-o/getsect.h>
 #include <mach-o/dyld_priv.h>
 #include <mach/i386/thread_status.h>
-#include <Availability.h>
 
 #include "FileAbstraction.hpp"
 #include "libunwind.h"
